examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/with-class-defined.py

```diff
@@ -19,18 +21,17 @@ class ShellScriptSchema(ResolvableSchema):
     asset_specs: Sequence[AssetSpecSchema]
 
 
+def resolve_asset_specs(
```

Comment:
another that seems like it should generated on the user's behalf

Reply:
these are all the same example fwiw -- it's just snapshotting it at different points in the user's creation of the component (so empty component, then adding schema, then parsing the schema, then building defs)